File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/Overgangsbelop.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.kode.FormelKodeCti;

import java.io.Serializable;

/**
 * @author Aasmund Nordstoga (Accenture) PK-5549
 */
public class Overgangsbelop implements Serializable {



    private FormelKodeCti formelKode;

    private int gpBrutto;

    private double gpSats;

    private double overgangsbelop;

    private int stBrutto;

    private int tpBrutto;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/Overgangsbelop.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.enum.FormelKodeEnum
import java.io.Serializable

class Overgangsbelop : Serializable {
    var formelKodeEnum: FormelKodeEnum? = null
    var gpBrutto = 0
    var gpSats = 0.0
    var overgangsbelop = 0.0
    var stBrutto = 0
    var tpBrutto = 0

}